src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
+const daysWithLowADiscount = 3;
+const daysWithLargeADiscount = 7;
+const rentOfEveryDay = 40;
+const lowDiscount = 20;
+const largeDiscount = 50;
```

Comment:
The variable names `daysWithLowADiscount`, `daysWithLargeADiscount`, `rentOfEveryDay`, `lowDiscount`, and `largeDiscount` are descriptive, but they could be simplified for better readability. Consider using names like `MIN_DAYS_FOR_DISCOUNT`, `MIN_DAYS_FOR_LARGE_DISCOUNT`, `DAILY_RENT`, `SMALL_DISCOUNT`, and `LARGE_DISCOUNT` respectively.